invenio_mail/ext.py

```diff
@@ -49,23 +49,33 @@ def print_email(message, app):
 
 
 class InvenioMail(object):
-    """Invenio-Mail extension.
+    """Invenio-Mail extension."""
 
-    Mails are only printed to the stream if ``MAIL_SUPPRESS_SEND`` is ``True``.
+    def __init__(self, app=None, stream=None):
+        """Extension initialization.
 
-    :param app: Flask application object.
-    :param stream: Stream to print emails to. Defaults to ``sys.stdout``.
-    """
+        Mails are only printed to the stream if ``MAIL_SUPPRESS_SEND`` is
+        ``True``.
 
-    def __init__(self, app=None, stream=None):
-        """Extension initialization."""
+        :param app: Flask application object.
```

Comment:
@JavierDelgadoFernandez make sure that you include `autoclass_content = 'both'` in `docs/conf.py`.